### PR TITLE
WEBUI-406: fetch correct version of chromedriver

### PR DESCRIPTION
**Pull request by Gabez0r**
The propose change allows the Web UI Functional Test framework to:
1) read the version of google chrome currently installed on the local machine
2) identify the ChromeDriver version that best works with the version of Chrome identified above and download it

The goal is to prevent us from having a mismatch between the local version of chrome and what the "latest" version of ChromeDriver is, which sometimes points to a version that only works with the latest beta of Chrome.

### CODE REVIEW

**Review by andreacornaglia — DISMISSED** [review]

File: packages/nuxeo-web-ui-ftest/scripts/test.js [hunk 1]
```diff
@@ -95,11 +98,52 @@ process.env.BROWSER = argv.browser || process.env.BROWSER || 'chrome';
...
+  let version;
+  try {
+    version = execSync(`"${chromePath}" --version`)
+      .toString()
+      .trim();
+  } catch (e) {
+    console.error('unable to get Chrome version: ', e);
+  }
+  // eslint-disable-next-line no-console
+  console.log(`${version} detected.`);
```
Comment (richardsd) [inline packages/nuxeo-web-ui-ftest/scripts/test.js:114]: Maybe `Chrome ${version} detected.` ?

**Review by richardsd — APPROVED** [review]

**Comment by richardsd** [comment]
> The propose change allows the Web UI Functional Test framework to:
> 
> 1. read the version of google chrome currently installed on the local machine
> 2. identify the ChromeDriver version that best works with the version of Chrome identified above and download it
> 
> The goal is to prevent us from having a mismatch between the local version of chrome and what the "latest" version of ChromeDriver is, which sometimes points to a version that only works with the latest beta of Chrome.

It looks good to me, don't forget to run it on legacy QA (just in case).

File: packages/nuxeo-web-ui-ftest/scripts/test.js (hunk 1 above)
Comment (Gabez0r) [reply packages/nuxeo-web-ui-ftest/scripts/test.js:114]: The version string also includes the product name. So "Google Chrome" will be already displayed.

Comment (richardsd) [reply packages/nuxeo-web-ui-ftest/scripts/test.js:114]: Ok, cool then 👍

**Review by mnixo — DISMISSED** [review]

**Review by semisse — APPROVED** [review]

**Comment by Gabez0r** [comment]
Force pushed to rebase on master and resolve conflicts.

**Review by mnixo — APPROVED** [review]

**Review by semisse — APPROVED** [review]

**Review by richardsd — APPROVED** [review]

**Pull request merged**